x-pack/legacy/plugins/apm/public/components/shared/TransactionActionMenu/TransactionActionMenu.tsx

```diff
@@ -210,18 +210,16 @@ export const TransactionActionMenu: FunctionComponent<Props> = (
     }
   ]
     .filter(({ condition }) => condition)
-    .map(({ icon, key, child, condition }) =>
-      condition ? (
-        <EuiContextMenuItem icon={icon} key={key}>
-          <EuiFlexGroup gutterSize="s">
-            <EuiFlexItem>{child}</EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <EuiIcon type="popout" />
-            </EuiFlexItem>
-          </EuiFlexGroup>
-        </EuiContextMenuItem>
-      ) : null
-    );
+    .map(({ icon, key, child }) => (
```

Comment:
This change looks good but not really related to the EUI update, is it?
Do leave it in though 👍

Reply:
`EuiContextMenuPanel` was converted to TS and its `items` prop no longer accepts `null` entries (it never really did, but it was typed). Removing the unneeded conditional returns a `null`-less array, which passes the type check.